stage3_emlid/07-ros/01-packages

```diff
@@ -1,5 +1,7 @@
-python-rosdep
-python-rosinstall-generator
-python-catkin-tools
+python3-rosdep
+python3-rosinstall-generator
+python3-catkin-tools
 python-wstool
-python-rosinstall
+python3-rosinstall
+python3-osrf-pycommon
+python3-cffi
```

Comment:
What is the rationale for adding new python packages?

Reply:
They seems to be needed packages that are not correctly stated as dependencies from the other ones, but requested at runtime.

Reply:
Can you post the runtime command and the error message that pops up?